common/djangoapps/third_party_auth/models.py

```diff
@@ -421,3 +427,60 @@ def current(cls, entity_id):
 
         cache.set(cls.cache_key_name(entity_id), current, cls.cache_timeout)
         return current
+
+
+class LTIProviderConfig(ProviderConfig):
+    """
+    Configuration required for this edX instance to act as a LTI
+    Tool Provider and allow users to authenticate and be enrolled in a
+    course via third party LTI Tool Consumers.
+    """
+    prefix = 'lti'
+    backend_name = 'lti'
+    icon_class = None  # This provider is not visible to users
+    secondary = False  # This provider is not visible to users
+    accepts_logins = False  # LTI login cannot be initiated by the tool provider
+    KEY_FIELDS = ('lti_consumer_key', )
+
+    lti_consumer_key = models.CharField(
+        max_length=255,
+        unique=True,
```

Comment:
You'll need to remove this line `unique=True` that you recently added. It prevents me from editing or disabling any `LTIProviderConfig`s:

![screen shot 2015-08-17 at 10 44 47 pm](https://cloud.githubusercontent.com/assets/945577/9323109/9559520c-4531-11e5-8e57-7c38047f3542.png)

The `ProviderConfig`/`ConfigurationModel` classes are designed to keep a history of the configuration in the table, and that means that if the configuration is edited, `lti_consumer_key` will not be unique within the table.